### ISSUE TITLE
add CAINFO to curl

### ISSUE THREAD

**Issue by yuyiguo** [dmwm/DBS#431]
This fix is from Dima.

here is what I had to fix in PycurlClient to make it work on SL6:
diff --git a/PycurlClient/src/python/RestClient/AuthHandling/X509Auth.py
b/PycurlClient/src/python/RestClient/AuthHandling/X509Auth.py
index 13b7399..e9ddec6 100644
--- a/PycurlClient/src/python/RestClient/AuthHandling/X509Auth.py
+++ b/PycurlClient/src/python/RestClient/AuthHandling/X509Auth.py
@@ -86,6 +86,7 @@ class X509Auth(object):
         curl_object.setopt(curl_object.CAPATH, self._ca_path)
         curl_object.setopt(curl_object.SSLCERT, self._ssl_cert)
         curl_object.setopt(curl_object.SSLKEY, self._ssl_key)
+        curl_object.setopt(curl_object.CAINFO, self._ssl_cert) # HACK
- FIX it properly

         if self.ssl_key_pass:
             curl_object.setopt(curl_object.SSLKEYPASSWD, self.ssl_key_pass)

Basically one needs to provide not only CAPATH, but CAINFO as well.
It's a bug/feature in curl. I used ssl_cert that works in my case, but
in general, it should be one more parameter that needs to be
configurable.

**Comment by yuyiguo** [comment]
Done.
